packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -0,0 +1,522 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React, {useState} from 'react'
+import {useProducts} from '@salesforce/commerce-sdk-react'
+import {findImageGroupBy} from '@salesforce/retail-react-app/app/utils/image-groups-utils'
+import {useCurrentCustomer} from '@salesforce/retail-react-app/app/hooks/use-current-customer'
+import ItemVariantProvider from '@salesforce/retail-react-app/app/components/item-variant'
+import {
+    Text,
+    Button,
+    Container,
+    Box,
+    Flex,
+    VStack,
+    HStack,
+    Image,
+    VisuallyHidden,
+    Select,
+    List,
+    ListItem
+} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {useIntl} from 'react-intl'
+import PropTypes from 'prop-types'
+
+const LOADING_LABEL = {
```

Comment:
These constants are only used once. How abut we move it directly into the code? Reason is these constant takes up most of the component implementation. Does not seem to have much values keeping them as constant here